addOns/automation/CHANGELOG.md

```diff
@@ -4,7 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
+### Changed
+- Maintenance changes.
```

Comment:
In a previous version we were explicit about that.

Reply:
Yeah, I waffled back and forth. Will adjust.

Reply:
Fixed